### DISCUSSION TITLE
Geometry doesn't show

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#12945] [Q&A]
Hi all,

I made a geometrry, but it doesn't show in smokesview. I don't know what i did wrong. I tested an example I found on github as well and that also did not show. Can someone look at my inpupt file if I did something wrong?

[test2.txt](https://github.com/firemodels/fds/files/15332990/test2.txt)

**Comment by rmcdermo** [comment]
Probably the version of your code, it is showing up fine for me.  What version of FDS are you using?
<img width="752" alt="Screenshot 2024-05-16 at 7 05 41 AM" src="https://github.com/firemodels/fds/assets/4418497/84018382-6ef3-4ed1-9c41-d2304e0e3eb2">

**Reply by bjmvandongen** [reply]
Thank you for your reply. I installed this file: FDS-6.9.1_SMV-6.9.1_lnx.sh.

**Comment by rmcdermo** [comment]
Please show the terminal output from your FDS run.

**Reply by bjmvandongen** [reply]
[TEST.sh.e214087.txt](https://github.com/firemodels/fds/files/15334672/TEST.sh.e214087.txt)

**Reply by bjmvandongen** [reply]
I figured it out! Not all the files in the folder were downloaded properly.... So, smokesview could not show the geometry.
